src/content/saas-vs-self-hosted.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: SaaS vs. Self-Hosted
+description: The difference between Okteto's product offerings in SaaS and Self-Hosted
+sidebar_label: SaaS vs. Self-Hosted
+id: saas-vs-self-hosted
+---
+
+Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the Okteto Platform. You can use the Okteto CLI without using the Okteto platform, but there are important limitations with that use case. 
```

Comment:
```suggestion
Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the Okteto Platform. You can use the Okteto CLI without using the Okteto Platform, but there are important limitations with that use case. 
```
Should we always use capital letters for `Okteto Platform`?

Reply:
I think so. At least for now until we coalesce on language. "Okteto Platform", to me, means all parts of Okteto that make the magic happen aside from the CLI. So that would include the Helm chart, web application/UI, etc. 

I missed this one (and probably others 😂) but happy to be convinced otherwise too if you or Ramiro feel different wording may be more appropriate